src/main/java/cart/controller/CartController.java

```diff
@@ -0,0 +1,49 @@
+package cart.controller;
+
+import cart.dto.ProductDto;
+import cart.service.CartService;
+import java.util.List;
+import javax.servlet.http.HttpServletRequest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/cart")
+public class CartController {
+
+    private final CartService cartService;
+
+    public CartController(CartService cartService) {
+        this.cartService = cartService;
+    }
+
+    @GetMapping("/all")
+    public ResponseEntity<Object> findAllProductInCart(HttpServletRequest request) {
+        int memberId = (int) request.getAttribute("memberId");
+
+        List<ProductDto> allProduct = cartService.findAllProduct(memberId);
+        return ResponseEntity.status(HttpStatus.OK).body(allProduct);
```

Comment:
```suggestion
        return ResponseEntity.ok(allProduct);
```

다른 status들도 이런 식으로 표현할 수 있습니다!

Reply:
개인적으로 status는 뭐고, body는 뭐고가 정확히 보인다는 측면에서
ResponseEntity.status(HttpStatus.OK).body(allProduct);가
낫다고 생각해서 해당 형식으로 통일했습니다..!ㅎㅎ